script/tasks/diamondMakeImmutable.sh

```diff
@@ -91,11 +86,10 @@ diamondMakeImmutable() {
   echo "PART 2 - SETTER FUNCTION REMOVAL IN FACETS"
   # remove all function selectors that will be unusable in the immutable diamond
   attempts=1
-  while [ $attempts -lt 11 ]
-  do
+  while [ $attempts -lt 11 ]; do
     echo "[info] >>>>>>>>>>>>>>>>>>>>>>>>>>>> Trying to remove unusable function selectors from (pre-)immutable diamond ($DIAMOND_ADDRESS) now - attempt ${attempts}"
     # try to execute call
-    RAW_RETURN_DATA=$(NETWORK=$NETWORK SALT="" FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=false PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") NO_BROADCAST=false forge script script/tasks/solidity/RemoveUnusableSelectorsFromImmutableDiamond.s.sol -f $NETWORK -vvvv --json --silent --broadcast --verify --skip-simulation --legacy --tc DeployScript)
+    RAW_RETURN_DATA=$(NETWORK=$NETWORK SALT="" FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=false PRIVATE_KEY=$(getAccount "$NETWORK" "$ENVIRONMENT") NO_BROADCAST=false forge script script/tasks/solidity/RemoveUnusableSelectorsFromImmutableDiamond.s.sol -f $NETWORK -vvvv --json --silent --broadcast --verify --skip-simulation --legacy --tc DeployScript)
```

Comment:
Seems like we dont even need to send an ENV-variable PRIVATE_KEY here since you updated the script to obtain the privKey from the storage, right?

(that might also apply to comments above)